modules/server/templates/scripts/forseti-server/run_forseti.sh.tpl

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+# Copyright 2020 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source /home/ubuntu/forseti_env.sh
+
+
+# set -x enables a mode of the shell where all executed commands are printed to the terminal.
+# With this  enabled, we should not put anything private/secret in the commands called because
+# they will be logged.
+set -x
+
+# Remove any CAI temp files generated by the forseti process.
+# Failed inventory due to various reasons could cause the temp files not being deleted.
+sudo rm -rf /tmp/forseti-cai-*
+
+# Put the config files in place.
+sudo gsutil cp gs://${storage_bucket_name}/configs/forseti_conf_server.yaml ${forseti_server_conf_path}
+sudo gsutil cp -r gs://${storage_bucket_name}/rules ${forseti_home}/
+
+# Download the Newest Config Validator constraints from GCS.
+if [ "${policy_library_sync_enabled}" != "true" ]; then
+  sudo gsutil cp -r gs://${storage_bucket_name}/policy-library ${policy_library_home}/
```

Comment:
I just created a [PR](https://github.com/forseti-security/terraform-google-forseti/pull/463) to fix an issue with this command on server reset and we should change it here as well. Running `gsutil cp` will only copy new/changed files from GCS to the VM and will not delete files on the VM that were deleted on GCS. Can you change this command to:
```
sudo gsutil -m rsync -d -r gs://$SCANNER_BUCKET/policy-library $POLICY_LIBRARY_HOME/policy-library
```

Reply:
Done